docker/build_scripts/build.sh

```diff
@@ -66,5 +66,11 @@ yum -y clean all > /dev/null 2>&1
 yum list installed
 
 for PYVER in $PY_VERS; do
-    /opt/$PYVER/bin/python $MY_DIR/manylinux1-check.py
+    soabi_flags_list="m"
+    if [ $(lex_pyver $PYVER) -lt $(lex_pyver 3.3) ]; then
+        soabi_flags_list="mu m"
+    fi
+    for soabi_flags in $soabi_flags_list; do
+        /opt/${PYVER}${soabi_flags}/bin/python $MY_DIR/manylinux1-check.py
+    done
```

Comment:
This should probably just be
```
for PYTHON in /opt/*/bin/python; do
    $PYTHON $MY_DIR/manylinux1-check.py
done
```
it's a bit redundant (checks the same interpreters repeatedly), but the test takes a fraction of a second so who cares -- being thorough and robust seems more important.